build-release-tools/lib/manifest-web-ui.json

```diff
@@ -0,0 +1,22 @@
+{
+    "build-name": "rackhd-devel", 
+    "build-requirements": "xenial", 
+    "downstream-jobs": [], 
+    "repositories": [
+        {
+            "branch": "",
+            "commit-id": "",
+            "repository": "https://github.com/RackHD/RackHD.git"
+        },
```

Comment:
Why add ```RackHD``` to the manifest file?
If the pipeline of on-web-ui checks "tag" or "jump version" , the action "jump version" or "create tag" will be apply to RackHD.
That will be duplicated with pipeline of RackHD...